Penteract/Assets/Scripts/Duke.cpp

```diff
@@ -44,7 +44,7 @@ void Duke::Init(UID dukeUID, UID playerUID, UID bulletUID, UID barrelUID, UID ch
 		bullet = bulletGO->GetComponent<ComponentParticleSystem>();
 		if (bullet) {
 			bullet->SetParticlesPerSecond(float2(0.0f, 0.0f));
-			bullet->Play();
+			//bullet->Play();
```

Comment:
@Mefiso  Should I remove this? If I don't, it will always shoot the first time because it's called from AIDuke's Start